lib/bolt/node/ssh.rb

```diff
@@ -19,8 +20,10 @@ def protocol
     end
 
     def connect
+      transport_logger = Logging.logger[Net::SSH]
+      transport_logger.level = :warn
```

Comment:
Hmm. Won't this just perpetually re-set the log level to `WARN` ever time a new connection is opened? What does one do if they need `DEBUG` logs for a connection issue?

Reply:
Yes, it will. That was the previous behavior so I preserved it while switching to the new logging library. Now that we have better facilities in place to control log level for these sorts of things, I think we need to do a pass through and figure out the behavior of `--verbose` and `--debug`. It definitely seems like it makes sense for everything to log at debug if specified.